marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/JobsControllerTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.hub.central.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.central.AbstractHubCentralTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JobsControllerTest extends AbstractHubCentralTest {
```

Comment:
Because this test has no dependency on ML or Spring, don't extend this base class. Just instantiate a new JobsController. That makes it clear that the method being tested has no dependencies too (and the test runs a bit faster too).

Reply:
The processTargetDatabase function in JobsController depends on Spring since it needs access to the HubClient for getting the database name.

Reply:
Ah - bummer